### PR TITLE
feat(stock): implement stock changes report

### LINKED ISSUE #5108: Feature Request:  Report to Record Changes in Inventory Value
To support stock in health zones, we'll want the health centers to bring in the quantities in stock for each product in stock so that we can record the consumption against them.  Similarly, when doing an "inventory" in your warehouse, the easiest way is to print out a copy of the current inventory with a column to allow you to record the correct values.  *It should look nearly identical to the Inventory Adjustment module*.

So, let's implement a report where you can pick...
  1. A depot
  2. A date

...and it will produce a report that should looks like the stock adjustment module, but with blank cells for the "new quantity".  This will allow users to print it, record changing values, and then key it in to the software later.

### PR DESCRIPTION
**Pull request by jniles**
Implements the "stock changes" report to allow users to download the
current inventory in a depot as of the end of a period.  This should
allow them to fill out the details and quickly compute the quantities in
stock to then make stock adjustments.

Closes #5108.

image

### CODE REVIEW

**Comment by jniles** [comment]
@lomamech can I get a review?

**Comment by lomamech** [comment]
@jniles Keys are missing for the translation
image

**Review by lomamech — COMMENTED** [review]
@jniles I emptied the Test database, and I carried out integrations for the month of January, suide consumption and losses, but I notice that the data on consumption and losses do not appear ?

image

File: client/src/i18n/en/report.json [hunk 1]
```diff
@@ -479,6 +479,10 @@
       "PAYMENTS" : "Payments",
       "AVERAGE_COST" : "Average Cost",
       "RECOVERY_CAPACITY" : "Recovery Capacity"
+    },
```
Comment (lomamech) [inline client/src/i18n/en/report.json:482]: These keys did not exist in the (fr) file

**Comment by lomamech** [comment]
:warning: :warning: :warning:Let me understand the purpose of this report is to print a document that will allow you to enter the situation in stock compared to the data of the month and by batches (Consumption, losses, days of breaks and remaining quantities) ????? :warning: :warning: :warning:

**Comment by jniles** [comment]
@lomamech thank you for the review. Yes, that is correct - the report should be blank so that users can download the report, print it out, and then be able to fill in the correct values over a month.

**Comment by lomamech** [comment]
> @lomamech thank you for the review. Yes, that is correct - the report should be blank so that users can download the report, print it out, and then be able to fill in the correct values over a month.

Add the keys for the missing translation to conclude this PR

File: client/src/i18n/en/report.json (hunk 1 above)
Comment (jniles) [reply client/src/i18n/en/report.json:482]: Great catch!  Thanks!

**Comment by jniles** [comment]
@lomamech, changes landed:

image

**Comment by jniles** [comment]
Bors r+

**Pull request merged**